src/land/smap_ssm2ioda.py

```diff
@@ -102,44 +107,42 @@ def _read(self):
             errs = errs[mask]
             qflg = qflg[mask]
             times = times[mask]
+            if sflg_present:
+                sflg = sflg[mask]
+                vegop = vegop[mask]
+                erowi = erowi[mask]
+                ecoli = ecoli[mask]
         # set fillValue to IODA missing
         vals[vals == _FillValue] = float_missing_value
 
-        # file provides yyyy-mm-dd as an attribute
-        # str_datetime = ncd.groups['Metadata'].groups['DatasetIdentification'].getncattr('creationDate')
-        # my_datetime = datetime.strptime(str_datetime, "%Y-%m-%d")
-        # get datetime from filename
-        file_refTime = re.search(r"\d{8}T\d{6}", self.filename).group()
-        file_refTime = datetime.strptime(file_refTime, "%Y%m%dT%H%M%S")
-        time_offset = round((file_refTime - epoch).total_seconds())
         vals = vals.astype('float32')
         lats = lats.astype('float32')
         lons = lons.astype('float32')
         deps = deps.astype('float32')
+        # reassign to contant (could be pass in)
```

Comment:
spelling typo: contant should be `constant`

Reply:
suggestion committed